plugins/flytekit-vscode/README.md

```diff
@@ -0,0 +1,40 @@
+# Flytekit VSCode Plugin
+
+The Flytekit VSCode plugin offers an easy solution for users to run tasks within an interactive VSCode server, compatible with any image. `@vscode` is a decorator which users can put within @task and user function. With `@vscode`, the task will install vscode dependencies and run a vscode server instead of the user defined functions.
+
+To install the plugin, run the following command:
+
+```bash
+pip install flytekitplugins-vscode
+```
+
+## Task Example
+```python
+from flytekit import task
+from flytekitplugins.vscode import vscode
+
+@task
+@vscode
+def train():
+    ...
+```
+
+## User Guide
+1. Run the decorated task on the remote. For example: `pyflyte run --remote [PYTHONFILE] [WORKFLOW|TASK] [ARGS]...`
+2. Once the code server is prepared, you can forward a local port to the pod. For example: `kubectl port-forward -n [NAMESPACE] [PODNAME] 8080:8080`.
+3. You can access the server by opening a web browser and navigating to `localhost:8080`.
+
+VSCode example screenshot:
+<img src="https://raw.githubusercontent.com/flyteorg/flytekit/master/plugins/flytekit-vscode/example.png">
```

Comment:
create `docs/` under flytekit-vscode folder and move `example.png` inside
you can refer the image by `<img src="./docs/example.png">`